scipy/special/functions.json

```diff
@@ -88,7 +88,8 @@
     },
     "bdtrin": {
         "_cdflib_wrappers.pxd": {
-            "bdtrin": "ddd->d"        }
+            "bdtrin": "ddd->d"
```

Comment:
Looks like my json auto formatter was active here without me noticing. The changes are good in my opinion though. Will leave it up to the reviewer, if I should remove them.